server/user_config.go

```diff
@@ -115,6 +119,29 @@ type UserConfig struct {
 	WebsocketCheckOrigin   bool            `mapstructure:"websocket-check-origin"`
 }
 
+// ToAllowCommandNames parse AllowCommands into a slice of CommandName
+func (u UserConfig) ToAllowCommandNames() ([]command.Name, error) {
+	var allowCommands []command.Name
+	var hasAll bool
+	for _, input := range strings.Split(u.AllowCommands, ",") {
+		if input == "all" {
```

Comment:
Shouldn't this block be moved after line 131 where `input  != ""` is checked?

Reply:
Thank you. The empty string might be better to handle by an early return. fixed.

https://github.com/runatlantis/atlantis/pull/2877/commits/2f1d0b3f3e6cb3cdc45066962e776abfad140e71